### ISSUE TITLE
Parameterise use of HTML module refs?

### ISSUE THREAD

**Issue by kristianmandrup** [jessitron/elm-rugs#11]
I've noticed all your examples tend to use fully expanded namespace refs such as:

`Html.Attributes.value` instead of importing `value` ref directly and then use/reference as `value`.

Would it be possible to parameterise some kind of preferences or would I have to create my own forked rug with my specific preferences? 

I assume there is a way for a rug to modify top level imports to always reflect each specific reference imported and used by a rug, such as `input`, `onClick` and `value` etc?
Basically, what are the best practices for this kind of scenario?

Cheers!
